### ISSUE TITLE
For components with `rich text` parameters, hyperlinks are not accessible with `Voice Over`, `Full Keyboard Access` and `Switch Control`

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#1684]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Pro

### Device OS version

iOS 27

### Is Liquid Glass disabled

- [ ] Yes, Liquid Glass is disabled (or OS version is older than 26)
- [x] No, Liquid Glass is enabled (or OS version is 26 or newer)

### OUDS iOS library version

develop (i.e. 3.0.0-beta.11+)

### Xcode version

Xcode 26.5 Build version 17F42

### Swift version

swift-driver version: 1.148.6 Apple Swift version 6.3.2 (swiftlang-6.3.2.1.108 clang-2100.1.1.101) Target: arm64-apple-macosx26.0

### App orientation

Portrait

### App language

English

### Device language

English

### Tokens libraries versions embedded in the app (check your logs)

Tokens v2.6

### Design System Toolbox app version (if used)

develop (i.e. 3.0.0-beta.11+)

### Design System Toolbox app build number (if used)

/

### Design System Toolbox app build type (if used)

debug

### Describe the issue

Some components exposes parameters with `AttributedString` type, for rich text mode.
However, if they receive a dedicated texts with hyperlinks inside, the hyperlinks are not accessible for:
- Voice Over
- Full Keyboard Access
- Switch Control

> [!NOTE]
> For Full Keyboard Access and Switch Control the hyperlinks are not reachable even if used in SwiftUI native text views.
> But hyperlinks are reachable for Voice Over if used inside Text

```swift
struct SandboxTestView: View {

    @State private var value: String = ""

    @Environment(\.theme) private var theme
    @Environment(\.colorScheme) private var colorScheme
    
    var body: some View {
        VStack {
            let richText = makeRichText()
            // OUDSPasswordInput(label: "Password", password: $value, helperText: richText)
            OUDSTextInput(label: "Label", text: $value, helperText: richText)
            OUDSHorizontalDivider()
            Text(richText)
        }
    }
    
    func makeRichText() -> AttributedString {
        let someRichtext = "Vous devez accepter les CGU et la PRIVACY policy"
        
        var configs: [AttributedStringUrlConfiguration] = []
        configs.append(AttributedStringUrlConfiguration(text: "CGU",
                                                        urlToOpen: URL("https://google.com"),
                                                        color: theme.colors.surfaceBrandPrimary.color(for: colorScheme),
                                                        font: Font.makeFont(family: theme.fontFamily,
                                                                            from: theme.fonts.bodyStrongLarge,
                                                                            isCompact: true).font))
        configs.append(AttributedStringUrlConfiguration(text: "PRIVACY",
                                                        urlToOpen: URL("https://orange.com"),
                                                        color: theme.colors.surfaceBrandSecondary.color(for: colorScheme),
                                                        font: Font.makeFont(family: theme.fontFamily,
                                                                            from: theme.fonts.bodyStrongLarge,
                                                                            isCompact: true).font))
        
        return AttributedString.from(text: someRichtext,
                                     foregroundColor: theme.colors.contentDefault.color(for: colorScheme),
                                     font: Font.makeFont(family: theme.fontFamily,
                                                         from: theme.fonts.labelDefaultLarge,
                                                         isCompact: true).font,
                                     urlConfigurations: configs)
    }
}
```

https://github.com/user-attachments/assets/50cae4f9-fdc8-4de0-b443-456a6d4e4931

### Expected Behavior

Let users be able to focus to open the hyperlinks with Voice Over, Fulk Keyboard Access and Switch Control

### Device settings

Voice Over, Full Keyboard Access, Switch Control
